gslib/addlhelp/creds.py

```diff
@@ -106,15 +106,16 @@
     For further information about account roles, see
     https://developers.google.com/console/help/#DifferentRoles
 
-  Google Compute Engine Internal Service Account:
+  Compute Engine Internal Service Account:
     This is the type of service account used for accounts hosted by App Engine
-    or Google Compute Engine. Such credentials are created automatically for
-    you on Google Compute Engine when you run the gcloud compute instances
-    creates command and the credentials can be controlled with the --scopes
+    or Compute Engine. Such credentials are created automatically for
+    you on Compute Engine when you run the "gcloud compute instances
```

Comment:
For code blocks, use double backticks (``) to begin and end the block

Reply:
Thanks, updated.